### ISSUE TITLE
[4.0] cassiopeia. Screenreader/Accessibility markup correct?

### ISSUE THREAD

**Issue by ReLater** [joomla/joomla-cms#26830]
### Steps to reproduce the issue
Is this markup correct?
https://github.com/joomla/joomla-cms/blob/4.0-dev/templates/cassiopeia/index.php#L104

On the one hand we have a `aria-hidden="true"` on the other several other "aria-" attributes like `aria-label`.

Maybe I just misunderstand the `aria-hidden` attribute but it looks illogical from my point of view.

**Comment by SharkyKZ** [comment]
You're right. `aria-hidden="true"` should be on the icon one line below.

**Comment by ReLater** [comment]
See pr https://github.com/joomla/joomla-cms/pull/26831
